src/index.ts

```diff
@@ -149,6 +155,61 @@ export const addKeypairToEnvFile = async (
   await appendFile(fileName, `\n${variableName}=${secretKeyString}`);
 };
 
+export interface initializeKeypairOptions {
+  envFileName?: string; // .env file path
```

Comment:
Please ditch the Javadoc-style comments. I think I mentioned this elsewhere but the idea is we use types to determine types, whether something is optional, etc. Use comments for situations that aren't obvious to a TS developer. This increases the signal:noise ratio of the code and reduces the chance of out of date comments misleading other developers.

Reply:
10-4